src/Console/CleanupVerificationCodesCommand.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace NextApps\VerificationCode\Console;
+
+use Illuminate\Console\Command;
+use NextApps\VerificationCode\Models\VerificationCode;
+
+class CleanupVerificationCodesCommand extends Command
+{
+    protected $signature = 'verification-code:cleanup {days : Codes older than these days will be deleted.}';
+
+    protected $description = 'Remove verification code older than the given days.';
```

Comment:
```suggestion
class PruneCommand extends Command
{
    protected $signature = 'verification-code:prune {--hours=24 : The number of hours to retain verification codes}';

    protected $description = 'Prune old verification codes';
```
I would set it up similarly to the prune command of Telescope (https://github.com/laravel/telescope/blob/4.x/src/Console/PruneCommand.php)